apps/tailwind-components/components/input/Decimal.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <InputString
+    :id="id"
+    :aria-describedby="describedBy"
+    :valid="valid"
+    :invalid="invalid"
+    :disabled="disabled"
+    :placeholder="placeholder"
+    :required="required"
+    v-model="modelValue"
+    @focus="$emit('focus')"
+    @blur="$emit('blur')"
+    @input="handleInputChanged"
+    @keypress="handleKeyValidity"
+  />
```

Comment:
I think we should throw an error if the user has only entered letters. I found it strange to be typing (not just numbers) and nothing happening.